README.md

```diff
@@ -456,6 +456,8 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [React Patterns](https://reactpatterns.com/)
 - [Useful Packages you should know in React](https://medium.com/better-programming/14-useful-packages-every-react-developer-should-know-55b47a325d3)
 - [Learn React Test Driven Development](https://learntdd.in/react/)
+- [React.js Cheatsheet](https://devhints.io/react)
```

Comment:
This resource is already added so please remove it

Reply:
Oh, sorry. I'm fixing it.